src/pdmat.jl

```diff
@@ -65,8 +65,8 @@ end
 
 ### quadratic forms
 
-quad(a::PDMat, x::StridedVector) = dot(x, a * x)
-invquad(a::PDMat, x::StridedVector) = dot(x, a \ x)
+quad(a::PDMat, x::AbstractVector) = dot(x, a * x)
```

Comment:
Looking through the internals of `quad!` and `invquad!` for the various types, all of the things that are called look safe to allow `AbstractArray`s, so that shouldn't be a problem. Thanks for your help here, @jmxpearson!